bitnami/spring-cloud-dataflow/templates/server/tls-secret.yaml

```diff
@@ -33,10 +30,6 @@ kind: Secret
 metadata:
   name: {{ $secretName }}
   namespace: {{ .Release.Namespace }}
-  labels: {{- include "common.labels.standard" . | nindent 4 }}
-    {{- if .Values.commonLabels }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
-    {{- end }}
```

Comment:
You completely removed the labels here

Reply:
fixed at https://github.com/bitnami/charts/pull/18749/commits/edc3ac67669f94c68712b8fe91c047ce6979a4e2